srcpkgs/ocs-url/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'ocs-url'
+pkgname=ocs-url
+version=3.1.0
+revision=1
+wrksrc="ocs-url-release-${version}"
+build_style="qmake"
+hostmakedepends="qt5-qmake qt5-svg kdeclarative qt5-svg-devel kdeclarative-devel qt5-declarative-devel"
```

Comment:
qt5-svg, kdeclarative are pulled in by -devel counterparts. qt5-qmake is not really needed in makedepends, please list needed ones, "duplication" is not bad here. Otherwise tested, works.

Reply:
I've made some edits, and hopefully it will still build after them
> ```sh
> hostmakedepends="qt5-qmake kdeclarative-devel qt5-svg-devel qt5-declarative-devel"
> makedepends="qt5-svg-devel kdeclarative-devel qt5-declarative-devel"
> ```

Reply:
checks passed, resolving!